.github/workflows/benchmarking.yml

```
@@ -343,6 +351,95 @@ jobs:
       env:
         SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
 
+  pgbench-pgvector:
+    strategy:
+      fail-fast: false
+
+    env:
+      TEST_PG_BENCH_DURATIONS_MATRIX: "5m"
```

Should we run it for a longer period to get more stable results?

---

ok, will increase this to 15m

---

It seems it is still 5 minutes 😄 

```suggestion
      TEST_PG_BENCH_DURATIONS_MATRIX: "15m"
```

---

oops. forgot to save the file locally. sorry for that